### PR TITLE
[ja] Localize Service Mesh for Japanese

### LINKED ISSUE #2768: [ja] Localize Service Mesh for Japanese
Localize https://glossary.cncf.io/service-mesh/

### PR DESCRIPTION
**Pull request by Okabe-Junya**
### Describe your changes

Localize Service Mesh for Japanese

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2768 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ja/service-mesh.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+サービスメッシュは、サービス間のトラフィック(すなわち通信)を管理することによって、この新しい課題に対処します。
+サービスメッシュは、すべてのサービスにわたって[信頼性](/ja/reliability/)、[可観測性](/ja/observability/)、およびセキュリティ機能を均一に追加します。
+
+## 解決すべき問題はなんですか
+
+マイクロサービスアーキテクチャに移行したことで、エンジニアは数百、
+場合によっては数千ものサービスを扱うようになり、それらすべてが相互に通信を必要としています。
+つまり、ネットワーク上で大量のトラフィックが行き交っています。
+その上、個々のアプリケーションは、必須の要件をサポートするために通信を暗号化する必要があるかもしれません。
+あるいは運用チームに共通のメトリクスを提供する必要があるかもしれませんし、問題の診断に役立つ、詳細なトラフィックの洞察を提供する必要があるかもしれません。
```
Comment (Okabe-Junya) [inline content/ja/service-mesh.md:19]: "detailed insight into traffic" を「詳細なトラフィックの洞察」と直訳してしまいましたが、少し不自然な感じがします。

Comment (naonishijima) [reply content/ja/service-mesh.md:19]: 「トラフィックの詳細な洞察」もしくは「トラフィックに関する詳細な洞察」はどうでしょうか。

Comment (Okabe-Junya) [reply content/ja/service-mesh.md:19]: > トラフィックに関する詳細な洞察

が最も適しているように感じました。ありがとうございます

**Review by naonishijima — APPROVED** [review]
LGTM

**Review by kaitoii11 — APPROVED** [review]
LGTM

**Pull request merged**